### ISSUE TITLE
Video Ads on iOS not working

### ISSUE THREAD

**Issue by ravio64344** [lilili87222/admob-for-flash#79]
Hi,
I am using this ane for admob and my banner ads and Interstitial ads working fine on iOS.
But i am getting below error when i try to call video ads function i.e. Admob.cacheVideo(video_id).

### :{"data":"Error Domain=com.google.ads Code=9 \"No ad returned from any ad server.\" UserInfo={NSLocalizedDescription=No ad returned from any ad server., NSLocalizedFailureReason=No ad returned from any ad server.}","instanceName":"defaultVideo","type":"onVideoLoadFail","bubbles":false,"cancelable":false,"target":null,"currentTarget":null,"eventPhase":2}

Help me to resolve this error.

Thank you.

**Comment by lilili87222** [comment]
No ad returned   . admob has no ad for show. load later again.

**Comment by ravio64344** [comment]
Hi,
Thank you for reply.
But i am trying to get video ads from last monday. But getting same results.

**Comment by lilili87222** [comment]
This error mean you have add video success.

**Comment by lilili87222** [comment]
Yes,you will get a fail message. admob has not enough video for show.

**Comment by ravio64344** [comment]
Hi,
I am not even getting test video ads. Is this the expected behaviour? It has be more than one month but we are not getting ads yet.
